CONTRIBUTING.md

```diff
@@ -153,24 +153,36 @@ WILD_IGNORE_SKIP=mold cargo test --features mold_tests
 WILD_IGNORE_SKIP=all cargo test --features external_tests
 ```
 
+## Commit messages
+
+The title of your commit message should ideally be written with a view to it being included in the
+release notes. i.e. write what would be useful for a user of the linker to read. Implementation
+details can go in the extended description.
+
+When we write the release notes, we try to categorise commits into groups. You can help by prefixing
+the commit title with a word that gives a clue as to which section it should go in. If you think it
+shouldn't go in the release notes, you can prefix the title with "chore:" or "typo:", neither of
+which get included in the notes.
```

Comment:
We could also filter by footer `changelog: skip` like in https://github.com/davidlattimore/wild/commit/921fa4b79a93311d52c6f7bc553126f903d993cc but that's up to you. Example of git-cliff config: https://github.com/davidlattimore/wild/pull/1256/files#diff-e1372c8b03c40942b5d828a90975054cb8aaed3b38189f434396f922ec41a584R90